src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -41,11 +42,13 @@ export const EDU_PAYMENT_INFORMATION_SAVE_SUCCEEDED =
 export const EDU_PAYMENT_INFORMATION_SAVE_FAILED =
   'EDU_PAYMENT_INFORMATION_SAVE_FAILED';
 
+const isProd = !environment.isProduction() ? 'cnp-' : '';
```

Comment:
I know this is going to be removed soon, but I think this should be named something like `cnp-prefix` or `dd-prefix`. `isProd` is almost a misleading name for what it is.

Reply:
I agree - I should not have taken that shortcut. Updated!